src/app/data-entries/buttonGroup/button-group-sample-5/button-group-sample-5.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+    selector: "app-button-group-sample-5",
+    styleUrls: ["./button-group-sample-5.component.scss"],
+    templateUrl: "./button-group-sample-5.component.html"
+})
+export class ButtonGroupSample5Component implements OnInit {
+
+    public rippleColor = "grey";
+
+    public displayDensity = "comfortable";
+    public displayDensities;
+
+    public ngOnInit() {
+        this.displayDensities = [
+            { label: "cosy", selected: this.displayDensity === "cosy", togglable: true },
```

Comment:
We should define the objects in ascending/descending order according to the density, so they get displayed in that order in the button-group as well. For example: "**compact**"->"**cosy**"->"**comfortable**"